### ISSUE TITLE
Walletconnect hangs in most places after scan qr code when connected with TOR

### ISSUE THREAD

**Issue by kdejaeger** [horizontalsystems/unstoppable-wallet-android#3876]
I have tried several places to connect my wallet and so far only uniswap works when connected to TOR.

1. https://roll.collab.land/connect?id=VTJGc2RHVmtYMTgvRyt0blNKMTNSZ21KbzFXU3RabUp4MXZNU1kwTXU5R0kyRXB5TEJzOWx3bEhhREwrVTY5QlE2QnhpWWdNOExQc0hNRHQrWHZoRnd2ZnZWbWVFaHdaelYrQTUxVlM1MVNNSGRKdVZ0ZzAzR1Z2ZU9TSWxmSkxnbVlFQ0t6VTNyOExlSDhlOWZBZXlmazJvMFdYaXcyaTY1UGo3WTczMzhEejhZeFVzQytvSENzSWlSaWFzaytN&callbackURL=https://prodred.collab.land/api/v3/wallet

2. Go to aurox on the desktop, login, click on scanner at the bottom, then on alerts, then choose walletconnect. Here's the link: https://getaurox.com/terminal/

3. https://trade.orionprotocol.io/swap -> add your wallet -> walletconnect
4. wanted to try 1inch ... hangs there as well: https://app.1inch.io/
5. yet another one hangs : https://app.sushi.com/swap

Never ending reload icon is shown.

**Comment by abdrasulov** [comment]
The remote bridge server refuses connection with the "403 Forbidden" error in the given examples. The only successful connection can be established only with Uniswap as you've mentioned.

I checked it with also other apps that support wallet-connect. To enable TOR I used the orbot. The result is the same. Apparently, the issue is on the wallet-connect bridge server side. Need to report it to the wallet-connect team
